modules/client-configuration/pages/clients-sle.adoc

```diff
@@ -26,9 +26,36 @@ The products you need for this procedure are:
 | {sles} 15 SP2
 | SUSE Linux Enterprise Server 15 SP2 x86_64
 
+| {sles} 15 SP3
+| SUSE Linux Enterprise Server 15 SP3 x86_64 (ALPHA)
```

Comment:
> This table raise some questions:
> 
> 1. at top we say: "The products you need for this procedure are:" . All? Do I really need all? No, only one of it. That what I want.

Good point, but I'm struggling to come up with an alternative. "Select the products you need for this procedure from this table"? "Use this table to determine which products you need for this procedure"?

> 2. We offer more "SLE" products, but we listed here only "SLES" (Server). What about SAP, HPC, Desktop, etc. ?

Yes, I think this chapter should rightly be called SLES. Unless we call it SLE, and say something at the top about "use these instructions for other SLE products, including SAP, HPC, etc ...". What do you think?

> 
> And should we really list "Alpha" products which cannot be seen by normal customers?

I'm not sure. I added it because it appeared in the list, and I know we listed it as supported in the relnotes for the last MU.

Reply:
> > This table raise some questions:
> > 
> > 1. at top we say: "The products you need for this procedure are:" . All? Do I really need all? No, only one of it. That what I want.
> 
> Good point, but I'm struggling to come up with an alternative. "Select the products you need for this procedure from this table"? "Use this table to determine which products you need for this procedure"?

"The following products apply for this procedure"  (maybe?)

> > 1. We offer more "SLE" products, but we listed here only "SLES" (Server). What about SAP, HPC, Desktop, etc. ?
> 
> Yes, I think this chapter should rightly be called SLES. Unless we call it SLE, and say something at the top about "use these instructions for other SLE products, including SAP, HPC, etc ...". What do you think?

What about to list the products which are currently under regular maintenance and a "footer" which say, that also older SP1 and other SLE products use this procedure.

I would list:

- SLES15 SP2
- SLES12 SP5
- SLES for SAP 15 SP2
- SLES for SAP 12 SP5

These are our main products. Desktop, HPC, RT, SUMA and older SPs I would mention with some words in a footer.

> > And should we really list "Alpha" products which cannot be seen by normal customers?
> 
> I'm not sure. I added it because it appeared in the list, and I know we listed it as supported in the relnotes for the last MU.

Well, we have now support for it in SUMA, but customers can only see it when they participate in the Beta Test and we ship a public beta or when the product gets released.

Reply:
> > > This table raise some questions:
> > > 
> > > 1. at top we say: "The products you need for this procedure are:" . All? Do I really need all? No, only one of it. That what I want.
> > 
> > 
> > Good point, but I'm struggling to come up with an alternative. "Select the products you need for this procedure from this table"? "Use this table to determine which products you need for this procedure"?
> 
> "The following products apply for this procedure" (maybe?)

That still sounds like you're saying all of them :thinking: 

> 
> > > 1. We offer more "SLE" products, but we listed here only "SLES" (Server). What about SAP, HPC, Desktop, etc. ?
> > 
> > 
> > Yes, I think this chapter should rightly be called SLES. Unless we call it SLE, and say something at the top about "use these instructions for other SLE products, including SAP, HPC, etc ...". What do you think?
> 
> What about to list the products which are currently under regular maintenance and a "footer" which say, that also older SP1 and other SLE products use this procedure.
> 
> I would list:
> 
> * SLES15 SP2
> * SLES12 SP5
> * SLES for SAP 15 SP2
> * SLES for SAP 12 SP5
> 
> These are our main products. Desktop, HPC, RT, SUMA and older SPs I would mention with some words in a footer.

Done.

> 
> > > And should we really list "Alpha" products which cannot be seen by normal customers?
> > 
> > 
> > I'm not sure. I added it because it appeared in the list, and I know we listed it as supported in the relnotes for the last MU.
> 
> Well, we have now support for it in SUMA, but customers can only see it when they participate in the Beta Test and we ship a public beta or when the product gets released.

OK, I'll comment them out for now, and we can add them back in when we got to beta.